python/tank/platform/qt/aspect_preserving_label.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2013 Shotgun Software Inc.
```

Comment:
What is the purpose of this class? I think it's something that is used in the login framework, where the code tries to cater for the need of multiple sized logos etc. I don't think we should add this to core.